app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
```

Comment:
- [ ] Please add the `foreign_key` attribute to the comments and likes associations as well to ensure that the owner of each record in those models is a User but referred to as an Author as per requirements. 🙏

Reply:
This change has been done well